config/configuration.rb

```diff
@@ -1,23 +1,115 @@
+require 'pathname'
+require 'dotenv'
+
 # dashboard app specific configuration
 class Configuration
-  class << self
-    attr_writer :app_development_enabled
-    attr_writer :app_sharing_enabled
-
-    # FIXME: temporary
-    attr_accessor :app_sharing_facls_enabled
-    alias_method :app_sharing_facls_enabled?, :app_sharing_facls_enabled
-
-    def app_development_enabled?
-      return @app_development_enabled if defined? @app_development_enabled
-      ENV['OOD_APP_DEVELOPMENT'].present? || DevRouter.base_path.exist?
-    end
-    alias_method :app_development_enabled, :app_development_enabled?
-
-    def app_sharing_enabled?
-      return @app_sharing_enabled if defined? @app_sharing_enabled
-      @app_sharing_enabled = ENV['OOD_APP_SHARING'].present?
-    end
-    alias_method :app_sharing_enabled, :app_sharing_enabled?
+  attr_writer :app_development_enabled
+  attr_writer :app_sharing_enabled
+
+  # FIXME: temporary
+  attr_accessor :app_sharing_facls_enabled
+  alias_method :app_sharing_facls_enabled?, :app_sharing_facls_enabled
+
+  # The app's configuration root directory
+  # @return [Pathname] path to configuration root
+  def config_root
+    Pathname.new(ENV["OOD_APP_CONFIG_ROOT"] || "/etc/ood/config/apps/dashboard")
+  end
+
+  def initializers_root
+    Pathname.new(ENV["OOD_APP_INITIALIZERS_ROOT"] || config_root.join("initializers"))
+  end
+
+  def load_external_config?
+    to_bool(ENV.fetch('OOD_LOAD_EXTERNAL_CONFIG', (rails_env == 'production')))
```

Comment:
More of a stylistic opinion, but this is also equivalent to:

```ruby
to_bool(ENV["OOD_LOAD_EXTERNAL_CONFIG"] || rails_env == "production)
```

So choose whichever you think is easiest to read.